index.html

```diff
@@ -4461,7 +4461,7 @@ <h3>
 The process of binding a <a>DID</a> to something in the real world, such as a
 person or a company, for example using <a>verifiable credentials</a> with the
 same subject as that <a>DID</a>, is out of scope for this specification. For
-more information, see the [[VC-DATA-MODEL]] instead.
+more information, see the [[VC-DATA-MODEL]].
```

Comment:
We should be consistent in writing out the spec name and then providing it's reference. While not always required, it would be nice to know what you're clicking through to (or if you even need to click). For example:

```
see the Verifiable Credentials Data Model [[VC-DATA-MODEL]].
```